test/e2e/antctl_test.go

```diff
@@ -33,6 +33,26 @@ type cmdAndReturnCode struct {
 	expectedReturnCode int
 }
 
+// TestAllAntctl is the top-level test which contains all subtests for
+// Antctl related test cases as they can share setup, teardown.
+func TestAllAntctl(t *testing.T) {
+	skipIfHasWindowsNodes(t)
+	skipIfHasEnvNotDefault(t)
+
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	t.Run("TestAllAntctl", func(t *testing.T) {
+		t.Run("testAntctlAgentLocalAccess", func(t *testing.T) { testAntctlAgentLocalAccess(t, data) })
```

Comment:
Why it needs create subtest of subtest? could it just be:
```
data, err := setupTest(t)
if err != nil {
	t.Fatalf("Error when setting up test: %v", err)
}
defer teardownTest(t, data)
        
t.Run("testAntctlAgentLocalAccess", func(t *testing.T) { testAntctlAgentLocalAccess(t, data) })
t.Run("testAntctlControllerRemoteAccess", func(t *testing.T) { testAntctlControllerRemoteAccess(t, data) })
...
```

Reply:
yes, you are right, I misunderstood the usage and add an extra subtest, will remove them.

Reply:
@lzhecheng @tnqn  looks like we have inconsistent code style for subtest, I am confused now, should I add an extra subtest again? looks like service_test.go is using extra subtest wrapper.

Reply:
An extra subtest wrapper is necessary when the subtests run in parallel. It's the boundary of "wait group".